Xamarin.PropertyEditing.Mac/Controls/BrushEditorControl.cs

```diff
@@ -124,6 +124,8 @@ protected override void UpdateValue ()
 			if (ViewModel.Solid != null) {
 				var title = GetTitle ();
 
+				this.popUpButton.ToolTip = ViewModel.Resource != null ? string.Format ("{0} - {1}: {2}", title, Properties.Resources.Resource, ViewModel?.Resource?.Name) : string.Empty;
```

Comment:
This should be something on the property button control itself, not something we have to keep replicating everywhere. As it stands this only shows up for solid brushes. See the windows version: https://github.com/xamarin/Xamarin.PropertyEditing/blob/master/Xamarin.PropertyEditing.Windows/PropertyButton.cs#L137

Reply:
Thanks!